utils/block_proto/app.js

```diff
@@ -19,31 +19,39 @@
  * @fileoverview The main entry of the block prototype maker.
  */
 
+// @ts-check
+
 import fs from 'fs';
```

Comment:
There is a jshint warning "'import' is only available in ES6 (use 'esversion: 6'). (W119)jshint(W119)" in my vscode environment. Same warning happens for 'const', 'of', etc. Need add config to eliminate this warning?

Reply:
For VS Code + JS development, jslint is kind of obsolete. E.g., there is a config line in the package.json:

"type": "module",

jslint should have taken this config line and turn ES6module mode on by default.

Use ESLint instead. ESLint is the recommended lint tool integrated with VS Code.